src/services/blocks/XCMDecoder.ts

```diff
@@ -0,0 +1,158 @@
+// Copyright 2017-2024 Parity Technologies (UK) Ltd.
+// This file is part of Substrate API Sidecar.
+//
+// Substrate API Sidecar is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import '@polkadot/api-augment';
+
+import { ApiPromise } from '@polkadot/api';
```

Comment:
Lets ensure we add the `type` for the import. We werent the best in the past about it but we should do it as much as we can moving forward as its good practice.

ie. `import type { ApiPromise } from '@polkadot/api`;

Reply:
I am sorry about this one! 🙏 It's on my notes but somehow it slips every time from my checks.